debian/changelog

```diff
@@ -1,3 +1,25 @@
+pi-top-4-miniscreen (5.0.0-2) bullseye; urgency=medium
+
+  * Add journal input
+  * [pre-commit.ci] pre-commit autoupdate (#169)
+  * Drop log_to_journal
+  * Don't cap versions if there's no reason to
+  * `pi-topd` (#172)
+  * [pre-commit.ci] pre-commit autoupdate (#171)
+  * New changelog entry: v5.0.0-1 (#170)
+  * Update .pre-commit-config.yaml
+  * Drop unused override_dh_installsystemd
+  * Use updated functions
+  * Only reset last active timer on state change
+  * Use short link
+  * [pre-commit.ci] pre-commit autoupdate (#175)
+  * No reloading of images (#176)
+  * [pre-commit.ci] pre-commit autoupdate (#180)
+  * [pre-commit.ci] pre-commit autoupdate (#184)
+  * Use logging instead of PTLogger
```

Comment:
```suggestion
  * Drop unused override_dh_installsystemd
  * Use updated functions
  * Only reset last active timer on state change
  * Use short link
  * No reloading of images (#176)
  * Use logging instead of PTLogger
  * [pre-commit.ci] pre-commit autoupdate (#175, #180, #184)
```